docs/blog/author.yaml

```diff
@@ -169,3 +169,7 @@
   avatar: avatars/nader-dabit.jpg
   twitter: "@dabit3"
   bio: Developer Advocate at AWS
+- id: Doug McDonald
+  avatar: avatars/doug-mcdonald.jpg
```

Comment:
avatar image is `.png` not `.jpg. Both formats are supported, but path need to be correct ;)

Reply:
Arg! Sorry on the case!